protos/schedules.proto

```diff
@@ -0,0 +1,28 @@
+syntax="proto3";
+
+package dataform.schedules;
+
+message Notification {
+  repeated string emails = 1;
+  bool success = 2;
+  bool failure = 3; 
+}
+
+message ScheduleOptions {
+  repeated string actions = 1;
+  bool includeDependencies = 2;
+  bool fullRefresh = 3;
+} 
+
+message Schedule {
+  string name = 1;
+  bool enabled = 2;
+  ScheduleOptions options = 3;
+  string cron = 4;
+  Notification notification = 5;
+}
+
+message Schedules {
```

Comment:
nit: Maybe let's call this `SchedulesJSON` just so it's really clear that this refers to a JSON file. Might also be worth a comment at the top of this file to say that field names cannot be changed as this is serialized as JSON.